source/deployment/webrtc.rst

```diff
@@ -0,0 +1,57 @@
+Video and Audio Calling (Beta)
+=====================================
+
+Mattermost supports an early preview of video and audio calling option using a self-hosted proxy. The implementation is based on `WebRTC integration <https://www.html5rocks.com/en/tutorials/webrtc/basics/>`_ enabling one-on-one video and audio chat without plugins.
+
+The feature is currently intended as a working prototype for community development and not recommended for production. It is supported on Chrome, Firefox and the Mattermost Desktop Apps.
```

Comment:
supported on Chrome, Firefox and the Mattermost Desktop Apps on a server running with SSL